monai/apps/datasets.py

```diff
@@ -51,6 +51,12 @@ class MedNISTDataset(Randomizable, CacheDataset):
             will take the minimum of (cache_num, data_length x cache_rate, data_length).
         num_workers: the number of worker threads to use.
             if 0 a single thread will be used. Default is 0.
+        progress: whether to display a progress bar.
```

Comment:
Would be misleading for the progress bar for downloading, could you review this?

Reply:
That's a good point! enhanced the description in the latest commit.

Thanks.